src/utilities/geometry/Plane.hpp

```diff
@@ -44,14 +44,11 @@ class Point3d;
 class Vector3d;
 
 /** Plane defines an infinite plane in 3D space.  The equation of a plane is
-   *  a*x + b*y + c*z = d, any point that satisfies this equation is on the plane.
+   *  a*x + b*y + c*z + d = 0, any point that satisfies this equation is on the plane.
```

Comment:
This incorrect comment made me loose hours. I couldn't understand why the results where incorrect even though I was sure my math was correct.